app/services/pii/signing_key.rb

```diff
@@ -0,0 +1,32 @@
+module Pii
+  class SigningKey
+    include Encodable
+
+    attr_accessor :rsa_key
+
+    DIGEST = 'SHA256'.freeze
+    @rsa_key = nil
```

Comment:
I started with a class variable `@@rsa_key` to avoid needing to re-read the key file each time the class was instantiated. Then reek told me to prefer a class instance variable instead. http://stackoverflow.com/questions/15773552/ruby-class-instance-variable-vs-class-variable